src/spot.sol

```diff
@@ -37,22 +37,22 @@ contract Spotter is LibNote {
 
     // --- Data ---
     struct Ilk {
-        PipLike pip;
-        uint256 mat;
+        PipLike pip;  // OSM
+        uint256 mat;  // Liquidation ratio [ray]
     }
 
     mapping (bytes32 => Ilk) public ilks;
 
-    VatLike public vat;
-    uint256 public par; // ref per dai
+    VatLike public vat; // CDP Engine
```

Comment:
nit: minimum of two spaces between code and start of comment

Reply:
Here and elsewhere I still question whether there's any point to labelling the `Vat` like this.